packages/react-code-editor/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -652,6 +653,7 @@ class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState> {
                 onChange={this.onChange}
                 onMount={this.editorDidMount}
                 theme={isDarkTheme ? 'vs-dark' : 'vs-light'}
+                loading={loading}
                 {...editorProps}
```

Comment:
`editorProps` should already allow `loading` to be passed. If we want `loading` to be a distinct property, we will need to add it to the props table.

Reply:
I missed that this already was in the props table and is unsed.